src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text.txt

```diff
@@ -234,4 +234,19 @@ Error({Kind:ErrorKind.Div0})
 "2/5/2023 12:00 AM"
 
 >> Text(TimeValue("15:06"))
-"3:06 PM"
\ No newline at end of file
+"3:06 PM"
+
+>> Text(Time(17,2,3), DateTimeFormat.LongTime, "vi-VN")
+"5:02:03 CH"
+
+>> Text(DateValue("02/27/2023"), DateTimeFormat.LongDate, "vi-VN")
+"27 Tháng Hai 2023"
+
+>> Text(DateTimeValue("02/27/2023"), DateTimeFormat.LongDateTime, "vi-VN")
+"27 Tháng Hai 2023 12:00:00 SA"
+
+>> Text(Boolean("True"), "", "vi-Vn")
+"true"
```

Comment:
is there ever a case where we don't return the literal string "true"? 
And can we double check that PowerApps returns the same values for the new cases.

Reply:
as offline chat: only different between our and PowerApps are https://github.com/microsoft/Power-Fx/issues/1120.